core/zoe/contracts/automaticRefund.js

```diff
@@ -20,14 +20,16 @@ export const makeContract = harden((zoe, terms) => {
         zoe.complete(harden([inviteHandle]), terms.assays);
         return `The offer was accepted`;
       },
-      makeInvite: makeSeatInvite,
-      getOffersCount: () => offersCount,
     });
     const { invite, inviteHandle } = zoe.makeInvite(seat);
     return invite;
   };
   return harden({
     invite: makeSeatInvite(),
+    publicAPI: {
```

Comment:
These are supposed to be query-only right? (By convention that is, since we do not (currently) have any way of enforcing that.) I would be disturbed if these were understood as a normal way for a contract to expose effect-causing methods, so let's document that should only be used for query ops, and only do so ourselves. By that criteria, this one is fine. Although perhaps we want to adopt a name instead of `publicAPI` that emphasizes that.

Reply:
I'm trying out something new, in that the publicAPI can be used to query and makeInvites if the invites aren't meant to be restricted. 

Is there a reason that you don't want to expose effect-causing methods, such as creating invites?

Reply:
Sorry, query-only was too narrow. Query-only + making invites is ok, where the *making* of the invite doesn't cause any interesting change of state by itself, and the capacity to change state can only be realized by redeeming the invite.

Not sure I can state the intuition, and not sure it's well motivated. But going beyond those bounds makes me itch. To be revisited later when I'm more focused.

Reply:
Ok, glad to clear that up. I think I am only planning to do query-only + making invites. If anything outside of that seems necessary, let's revisit that